src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -191,6 +191,7 @@ def get_pairs(word):
     "ba": "bashkir",
     "jw": "javanese",
     "su": "sundanese",
+    "yue": "cantonese",
```

Comment:
Changes here should not affect the code as we use `index` and otherwise access the mapping.

Reply:
hmmmm..... as we're >= py3.8 this is technically fine but a bit cheeky!